identities/tasks.py

```diff
@@ -21,3 +24,23 @@ def run(self, anum, **kwargs):
         return int(anum)+1
 
 the_incr = The_Incr()
+
+
+class DeliverHook(Task):
+    def run(self, target, payload, instance=None, hook=None, **kwargs):
+        """
+        target:     the url to receive the payload.
+        payload:    a python primitive data structure
+        instance:   a possibly null "trigger" instance
+        hook:       the defining Hook object
+        """
+        requests.post(
+            url=target,
+            data=json.dumps(payload),
+            headers={
+                'Content-Type': 'application/json',
+                'Authorization': 'Token %s' % settings.HOOK_AUTH_TOKEN
```

Comment:
@imsickofmaps Our task has the optout instance, which has a link to the record for the requesting service, which can have a password added to it, so we don't need to extend anything unusual? Unless you mean that because we're using the Django user as our permission system, we'll need to link something to the Django users? We can just create a model that references the user table like we do for profiles?